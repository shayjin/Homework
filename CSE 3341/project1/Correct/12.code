pRogram Read wriTe iF TheN ENDIF BEGIN EnD reference EOS