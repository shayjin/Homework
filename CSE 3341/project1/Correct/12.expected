ID[pRogram]
ID[Read]
ID[wriTe]
ID[iF]
ID[TheN]
ID[ENDIF]
ID[BEGIN]
ID[EnD]
REF
ID[EOS]
